ansys/dpf/core/operators/averaging/elemental_difference.py

```diff
@@ -1,248 +1,313 @@
-"""Operators from Ans.Dpf.FEMutils plugin, from "averaging" category
-"""
 
 class elemental_difference(Operator):
-    """Transform ElementalNodal or Nodal field into Elemental field. Each elemental value is the maximum difference between the computed result for all nodes in this element. Result is computed on a given element scoping.
-
-      available inputs:
-        - field (Field, FieldsContainer)
-        - mesh_scoping (Scoping) (optional)
-        - mesh (MeshedRegion) (optional)
-        - through_layers (bool) (optional)
-
-      available outputs:
-        - fields_container (FieldsContainer)
-
-      Examples
-      --------
-      >>> from ansys.dpf import core as dpf
-
-      >>> # Instantiate operator
-      >>> op = dpf.operators.averaging.elemental_difference()
-
-      >>> # Make input connections
-      >>> my_field = dpf.Field()
-      >>> op.inputs.field.connect(my_field)
-      >>> my_mesh_scoping = dpf.Scoping()
-      >>> op.inputs.mesh_scoping.connect(my_mesh_scoping)
-      >>> my_mesh = dpf.MeshedRegion()
-      >>> op.inputs.mesh.connect(my_mesh)
-      >>> my_through_layers = bool()
-      >>> op.inputs.through_layers.connect(my_through_layers)
-
-      >>> # Instantiate operator and connect inputs in one line
-      >>> op = dpf.operators.averaging.elemental_difference(field=my_field,mesh_scoping=my_mesh_scoping,mesh=my_mesh,through_layers=my_through_layers)
-
-      >>> # Get output data
-      >>> result_fields_container = op.outputs.fields_container()"""
-    def __init__(self, field=None, mesh_scoping=None, mesh=None, through_layers=None, config=None, server=None):
-        super().__init__(name="elemental_difference", config = config, server = server)
+    """Transform ElementalNodal or Nodal field into Elemental field. Each
+    elemental value is the maximum difference between the computed
+    result for all nodes in this element. Result is computed on a
+    given element scoping.
+
+    Parameters
+    ----------
+    field : Field or FieldsContainer
+        Field or fields container with only one field
+        is expected
+    mesh_scoping : Scoping, optional
+        Average only on these entities
+    mesh : MeshedRegion, optional
+    through_layers : bool, optional
+        The max elemental difference is taken through
+        the different shell layers if true
+        (default is false)
+
+
+    Examples
+    --------
+    >>> from ansys.dpf import core as dpf
+
+    >>> # Instantiate operator
+    >>> op = dpf.operators.averaging.elemental_difference()
+
+    >>> # Make input connections
+    >>> my_field = dpf.Field()
+    >>> op.inputs.field.connect(my_field)
+    >>> my_mesh_scoping = dpf.Scoping()
+    >>> op.inputs.mesh_scoping.connect(my_mesh_scoping)
+    >>> my_mesh = dpf.MeshedRegion()
+    >>> op.inputs.mesh.connect(my_mesh)
+    >>> my_through_layers = bool()
+    >>> op.inputs.through_layers.connect(my_through_layers)
+
+    >>> # Instantiate operator and connect inputs in one line
```

Comment:
I know these are all autogenerated, but in the future comments should be on their own line without `>>> #` with whitespace between:

```suggestion
    Instantiate operator and connect inputs in one line

```